candid/evm_rpc.did

```diff
@@ -81,6 +81,14 @@ type HttpOutcallError = variant {
 type InstallArgs = record {
   demo : opt bool;
   manageApiKeys : opt vec principal;
+  consoleFilter : opt ConsoleFilter;
```

Comment:
Renamed `logMessageFilter` to `consoleFilter`, since this only affects the dfx console output and not the canister logs.

Reply:
good catch!